src/Microsoft.AspNet.Diagnostics/Views/BaseView.cs

```diff
@@ -52,7 +53,7 @@ public abstract class BaseView
             Context = context;
             Request = Context.Request;
             Response = Context.Response;
-            Output = new StreamWriter(Response.Body);
+            Output = new StreamWriter(Response.Body, Encoding.UTF8, 1024, leaveOpen: true);
```

Comment:
hmm, we don't set the charset in the content-type header do we? Can you add that while you're here?

Also, raise the buffer to 4kb. These pages are pretty big.